services/user-tenant-service/src/main/java/com/sourcefuse/jarc/services/usertenantservice/repository/TenantRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.sourcefuse.jarc.services.usertenantservice.repository;
+
+import com.sourcefuse.jarc.services.usertenantservice.dto.Tenant;
+import java.util.UUID;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface TenantRepository extends JpaRepository<Tenant, UUID> {}
```

Comment:
This needs to be implemented using soft delete repo if its being consumed in the node js catalog